src/app/_components/RecentBlocks/consts.ts

```diff
@@ -0,0 +1,4 @@
+export const BTC_BLOCK_MIN_WIDTH = '184px';
```

Comment:
put in sizes.ts

Reply:
Same for these widths, I think it would help to establish some guidelines around what's considered a design system constant vs a contextual value.